gnome-shell/src/gnome-shell-sass/_colors.scss

```diff
@@ -11,8 +11,8 @@ $selected_bg_color: if($variant == 'light', $orange, darken($orange, 4%));
 $selected_borders_color: if($variant== 'light', darken($selected_bg_color, 15%), darken($selected_bg_color, 30%));
 $borders_color: if($variant == 'light', darken($bg_color, 20%), darken($bg_color, 12%));
 $borders_edge: if($variant == 'light', transparentize(white, 0.2), transparentize($fg_color, 0.93));
-$link_color: $blue; 
-$link_visited_color: darken($blue, 5%);
+$link_color: if($variant == 'light', darken($selected_bg_color, 10%), lighten($selected_bg_color, 20%));
+$link_visited_color: if($variant == 'light', darken($selected_bg_color, 20%), lighten($selected_bg_color, 10%));
```

Comment:
Yeah this is what we use in gtk3. But shouldn't the link visited color be somehow related to the link color? Why should the link switch from blue to orange after clicking? :thinking:

Reply:
makes sense